schemas/IlpHeader.json

```diff
@@ -6,11 +6,11 @@
   "properties": {
     "ledger": {
       "description": "The ledger where the transfer will take place",
-      "$ref": "Iri.json"
+      "type": "string"
     },
     "account": {
       "description": "Account holding the funds",
-      "$ref": "Iri.json"
+      "type": "string"
```

Comment:
Undecided - the main two options are:

* Any OCTET STRING, i.e. no restriction at all - in that case the JSON version would likely be base64url encoded, so you'd have [a-zA-Z0-9_-] as the regex.
* IA5 (aka ASCII) or some subset.

For right now, let's just go with [url-safe characters](https://tools.ietf.org/html/rfc3986#section-2.3) - `/[a-zA-Z0-9._~-]*/`.